pandas/io/excel.py

```diff
@@ -156,11 +159,18 @@ def get_writer(engine_name):
         Acceptable values are None or xlrd"""
 
 @Appender(excel_doc_common % read_excel_kwargs)
```

Comment:
Yes - `ExcelFile.parse` and `read_excel` both use it.  

Just thinking, maybe the api should be changed to match the `to_excel` semantics where an `ExcelFile` object could be passed to `read_excel` (and deprecate `ExcelFile.parse`).